Source/Immutable/Private/Immutable/ImtblBrowserWidget.cpp

```diff
@@ -78,26 +66,19 @@ void UImtblBrowserWidget::ExecuteJS(const FString& ScriptText) const
 void UImtblBrowserWidget::SetBrowserContent()
 {
 #if USING_BUNDLED_CEF
-	FSoftObjectPath AssetRef(TEXT("/Script/Immutable.ImtblSDKResource'/Immutable/" "PackagedResources/index.index'"));
-	if (UObject* LoadedAsset = AssetRef.TryLoad())
+	if (!WebBrowserWidget.IsValid())
 	{
-		if (auto Resource = Cast<UImtblSDKResource>(LoadedAsset))
-		{
-			if (!WebBrowserWidget.IsValid())
-			{
-				IMTBL_ERR("no browser")
-				return;
-			}
-
-			const FString IndexHtml = FString("<!doctype html><html lang='en'><head><meta " "charset='utf-8'><title>GameSDK Bridge</title><script>") + Resource->Js + FString("</script></head><body><h1>Bridge Running</h1></body></html>");
-
-			// IMTBL_LOG("Loaded resource: %s", *Resource->GetName())
-			WebBrowserWidget->LoadString(IndexHtml, TEXT("file:///immutable/index.html"));
-			// WebBrowserWidget->LoadURL(FString::Printf(TEXT("%s%s"),
-			// TEXT("file:///"),
-			// *FPaths::ConvertRelativePathToFull(FPaths::Combine(FPaths::ProjectContentDir(),
-			// TEXT("html"), TEXT("index.html")))));
-		}
+		IMTBL_ERR("Browser widget is not valid")
+		return;
+	}
+	
+	FString JavaScript;
+	
+	if (FImmutableUtilities::LoadGameBridge(JavaScript))
```

Comment:
What happens if the SDK can't load the game bridge? Does it throw an error?

Reply:
I have updated log messages in `ImmutableUtilities.cpp`

Reply:
I don't think logging is sufficient. Shouldn't it throw a proper error? If it can't load the game bridge file, users won't be able to use our SDK anyway.

Reply:
I logged [a ticket](https://immutable.atlassian.net/browse/DX-3039). It is out of scope for this issue.